src/os/bluestore/BlueStore.cc

```diff
@@ -4886,36 +4886,47 @@ void BlueStore::_init_logger()
 		 "Average kv_thread flush latency",
 		 "fl_l", PerfCountersBuilder::PRIO_INTERESTING);
   b.add_time_avg(l_bluestore_kv_commit_lat, "kv_commit_lat",
-		 "Average kv_thread commit latency");
+		 "Average kv_thread commit latency",
+		 "co_l", PerfCountersBuilder::PRIO_USEFUL);
   b.add_time_avg(l_bluestore_kv_sync_lat, "kv_sync_lat",
 		 "Average kv_sync thread latency",
 		 "ks_l", PerfCountersBuilder::PRIO_INTERESTING);
   b.add_time_avg(l_bluestore_kv_final_lat, "kv_final_lat",
 		 "Average kv_finalize thread latency",
 		 "kf_l", PerfCountersBuilder::PRIO_INTERESTING);
   b.add_time_avg(l_bluestore_state_prepare_lat, "state_prepare_lat",
-    "Average prepare state latency");
+    "Average prepare state latency",
+    "sp_l", PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
May I suggest a single format for bluestore_state_*_lat: s??l, e.g. sprl for state_prepare, and sidl for stte_io_done_lat...

Similarly one might have k??l for for kv_*latency, e.g. kcol for kv_commit_lat...